Code/Framework/AzCore/AzCore/Serialization/AZStdContainers.inl

```diff
@@ -813,7 +893,7 @@ namespace AZ
             }
 
             /// Get an element's address by its key. Not used for serialization.
-            void*   GetElementByKey(void* instance, const SerializeContext::ClassElement* classElement, const void* key) override
+            void*   GetElementByKey(void* instance, const SerializeContext::ClassElement* classElement, const void* key) const override
```

Comment:
Nit:
```suggestion
            void* GetElementByKey(void* instance, const SerializeContext::ClassElement* classElement, const void* key) const override
```

Reply:
I'll defer fixing the spacing here, until another change is made to AZStdContainers.inl